src/gui/tray/Window.qml

```diff
@@ -405,6 +464,13 @@ Window {
                         UserModel.openCurrentAccountServer();
                     }
 
+                    Accessible.role: Accessible.ButtonMenu
+                    Accessible.name: qsTr("More apps")
+                    Accessible.onPressAction: {

```

Comment:
Commenting here but that applies to all the `Accessible.onPressAction` in here. They consistently duplicate the code from an `onClick` or similar. Could we make sure this duplication doesn't occur? Can't we just invoke the other signal handler? If not probably worth introducing an intermediate function as last resort.